File: problem-test/src/main/java/com/dudgns/problemtest/entity/carManagement/CompanyEntity.java
package com.dudgns.problemtest.entity.carManagement;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Builder
@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "company")
public class CompanyEntity {
    @Id
    private Integer companyCode;

    @Column
    private String companyName;
}

File: problem-test/commons/project-dudgns-data-entity/src/main/kotlin/dudgns/com/backend/commons/data/entity/problemTest/CompanyEntity.kt
package dudgns.com.backend.commons.data.entity.problemTest

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "company")
class CompanyEntity {
    @Id
    @Column(name = "company_code")
    var companyCode: Int? = null

    @Column(name = "company_name")
    var companyName: String? = null
}